src/pages/distroSettings/DistroSelect/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { useQuery } from "@apollo/client";
+import { useNavigate } from "react-router-dom";
+import SearchableDropdown from "components/SearchableDropdown";
+import { DistrosQuery, DistrosQueryVariables } from "gql/generated/types";
+import { GET_DISTROS } from "gql/queries";
+
+interface DistroSelectProps {
+  getRoute: (distroId: string) => string;
+  selectedDistro: string;
+}
+
+export const DistroSelect: React.VFC<DistroSelectProps> = ({
+  getRoute,
```

Comment:
[nit] I don't think `getRoute` needs to be a prop since this selector is less general than the project selector

Reply:
![facepalm-crowd](https://github.com/evergreen-ci/spruce/assets/47064971/e97dfacd-b073-43ad-a7df-e99c4d4ec048)